### DISCUSSION TITLE
MacOS third-party applications

### DISCUSSION THREAD

**Discussion by soutienti** [SafeExamBrowser/seb-mac#375] [Q&A]
Hello
Do you know when the MacOS version of SEB will support third-party applications ?

this is to be able to add the tools used by students with special needs during exams. (software that read text, helps dyslexia, etc)
We had software in the windows version and it's so useful, intuitive and simple.
Thanks a lot for your help. 
Safe examen is really a amazing project for school.

**Comment by danschlet** [comment]
There is no timeline, it's in the roadmap as the label indicates. It could be added to version 3.5, which might be released around end of the summer.

**Reply by soutienti** [reply]
thanks for your answer.

**Comment by danschlet** [comment]
See https://github.com/SafeExamBrowser/seb-mac/discussions/418
